.rubocop_todo.yml

```diff
@@ -6,6 +6,14 @@
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: Max, AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, AllowedPatterns.
+# URISchemes: http, https
+Layout/LineLength:
+  Exclude:
+    - 'spec/system/admin/tag_rules_spec.rb'
+
```

Comment:
Do you think that it would be worth re-running rubocop --regenerate-todo before commit so that new violations are not listed in the next commit? It would slow down the process but it would create a better commit history and is automatic.

Or we stop the script if there are new violations so that they can be fixed first.

Reply:
I think re-running would be the best option. Like you said earlier, if you start a script, you want it to just run by itself and not stop a minute into it for more user input.

Patching up the lingering errors in a final PR shouldn't be too big of a deal.